tests/infrared/16/infrared-openstack.sh

```diff
@@ -11,29 +11,33 @@ AMQP_PORT=${AMQP_PORT:-443}
 SSH_KEY="${SSH_KEY:-${HOME}/.ssh/id_rsa}"
 NTP_SERVER="${NTP_SERVER:-clock.redhat.com,10.5.27.10,10.11.160.238}"
 
-VM_IMAGE_URL_PATH="${VM_IMAGE_URL_PATH:-http://127.0.0.1/my_image_location/}"
-if [ "${VM_IMAGE_URL_PATH}" = "http://127.0.0.1/my_image_location/" -a -z "${VM_IMAGE}" ]; then
-    echo "Please provide a VM_IMAGE_URL_PATH or VM_IMAGE"
-    exit 1
-fi
+VM_IMAGE_URL_PATH="${VM_IMAGE_URL_PATH:-http://download.eng.bos.redhat.com/brewroot/packages/rhel-guest-image/8.1/333/images}"
 # Recommend these default to tested immutable dentifiers where possible, pass "latest" style ids via environment if you want them
-VM_IMAGE="${VM_IMAGE:-${VM_IMAGE_URL_PATH}/rhel-guest-image-8.1-413.x86_64.qcow2}"
-OSP_BUILD="${OSP_BUILD:-RHOS_TRUNK-16.0-RHEL-8-20200204.n.1}"
+VM_IMAGE="${VM_IMAGE:-rhel-guest-image-8.1-333.x86_64.qcow2}"
```

Comment:
Interesting that this appears to be going backwards. As long as you're sure. :)

Reply:
I only picked what magic combination worked for me based on what QE is testing in Jenkins. Since this is the base image in the Phase 1 that QE is testing with, I just aligned to it regardless of my thoughts :)

(It's possible the job is wrong, but I'm not sure how to determine that.)